students/chieu_quach/lesson03/mailroom.py

```diff
@@ -0,0 +1,157 @@
+
+# Author     -  Chieu Quach
+# Assignment -  Lesson 3
+# Exercise   -  Mailroom Part 1
+
+import sys
+global donation_list, len_donation_list
+donation_list = {}
+donation_list = [("William Gates III", 653784.49, 2, 326892.24),
+                 ("Mark Zukerberg", 16396.10, 3, 5465.37),
+                 ("Jeff Bezos", 877.33, 1, 877.33),
+                 ("Paul Allen", 708.42, 3, 236.14),
+                ]
+         # donation_list [] [] - to access name enter [1][0]
+         # the name (like Bill Gates) locates at [0]
+
+len_donation_list = len(donation_list)
+prompt = "\n".join(("Welcome to the Mailroom Part 1",
+                    "Please choose from below options: ",
+                    "1 - Send a Thank You",
+                    "2 - Create a Report",
+                    "3 - Quit",
+                    ">>>"))
+
+from operator import itemgetter
```

Comment:
You'll want to move this to the beginning of your script, along with the other import statements